scripts/build_release.py

```diff
@@ -39,6 +39,9 @@ def build_linux():
     maybe_remove(out)
     shutil.move(out_pyi, out)
 
+    # prevents missing export error on ubuntu 16.04 with intel graphics
+    maybe_remove(os.path.join(out, 'libdrm.so.2'))
```

Comment:
looks like you'll need to update the maybe_remove function to let it delete files in addition to directories.

Reply:
bah, do you have some code that will work?

Reply:
nvm, I just changed it to `os.remove` since it should always exist anyway